main/active_cname.js

```diff
@@ -21,147 +21,142 @@
  * CLOUDFLARE:  JS.COOL uses CloudFlare as its DNS. By default, CloudFlare proxies all requests to your subdomain
  *              to get SSL support (https://foo.js.cool) and make use of browser caching with a TTL of 30 min.
  *              But you can opt-out from this and make Cloudflare forward all requests directly to GitHub.
- *              Just add '//noCF' in the line of your requested subdomain to give us a hint.
- *              (all the lines marked with '//noCF?' are from a time when a requester had to explicitly opt-in.
- */
+ *              Just add "// noCF" in the line of your requested subdomain to give us a hint.
+ *              (all the lines marked with "// noCF?" are from a time when a requester had to explicitly opt-in.
+*/
+
 module.exports = {
-  // Cloudflare CDN
-  'github': 'cdn-github.willin.workers.dev', // github.com
-  'fonts-gstatic': 'cdn-fonts-gstatic.willin.workers.dev', // fonts.gstatic.com
-  'remix-i18n': 'remix-i18n.pages.dev',
-  // 请在此处区域内添加自定义域名
```

Comment:
?

Reply:
It is clear to people I believe to put their domains here